### PR TITLE
Add TGA load script

### PR DESCRIPTION
**Pull request by KingYiuWoo**

### CODE REVIEW

File: dle/data/management/commands/load_tga_data.py [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+        "keep track of the number of labels processed"
+        self.error_urls = {}
+        "dictionary to keep track of the urls that have parsing errors; form: {url: True}"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--type",
+            type=str,
+            help="'full'",
+            default="full",
```
Comment (ColeDCrawford) [inline dle/data/management/commands/load_tga_data.py:51]: Do we want to add tests at this point? E.g. https://github.com/globalrx/globalrx/blob/27871ed2e56eb6935bc4aaacd0d27974892fc3b1/dle/data/tests.py#L81-L86

Comment (KingYiuWoo) [reply dle/data/management/commands/load_tga_data.py:51]: Maybe wait til we have the parse support?

Comment (KingYiuWoo) [reply dle/data/management/commands/load_tga_data.py:79]: https://cscie599-drug-labels.atlassian.net/browse/DL-107
Created a ticket to add tests

File: dle/data/models.py [hunk 2]
```diff
@@ -21,7 +21,7 @@ class DrugLabel(models.Model):
 
     source = models.CharField(max_length=8, choices=SOURCES, db_index=True)
     product_name = models.CharField(max_length=255, db_index=True)
-    generic_name = models.CharField(max_length=255, db_index=True)
+    generic_name = models.CharField(max_length=2048, db_index=True)
```
Comment (ColeDCrawford) [inline dle/data/models.py:24]: Did this generate a migration?

**Comment by ColeDCrawford** [comment]
Sweet thanks for adding that migration, merging

**Review by ColeDCrawford — APPROVED** [review]

**Pull request merged**